samples/ingestion/ingestion-client/Setup/ArmTemplateBatch.json

```diff
@@ -1,233 +1,209 @@
+      "defaultValue": "",
+      "type": "String",
+      "metadata": {
+        "description": "The id of the custom model for transcription. If empty, the base model will be selected."
+      }
     },
+    "AzureSpeechServicesKey": {
+      "type": "SecureString",
+      "metadata": {
+        "description": "The key for the Azure Speech Services subscription."
+      }
+    },
+    "AzureSpeechServicesRegion": {
+      "defaultValue": "westus",
+      "type": "String",
+      "allowedValues": [
+        "centralus",
+        "eastus",
+        "eastus2",
+        "northcentralus",
+        "southcentralus",
+        "westcentralus",
+        "westus",
+        "westus2",
+        "canadacentral",
+        "brazilsouth",
+        "eastasia",
+        "southeastasia",
+        "australiaeast",
+        "centralindia",
+        "japaneast",
+        "japanwest",
+        "koreacentral",
+        "northeurope",
+        "westeurope",
+        "francecentral",
+        "uksouth",
+        "usgovarizona",
+        "usgovvirginia"
+      ],
+      "metadata": {
+        "description": "The region the Azure speech services subscription is associated with."
+      }
+    },
+    "CustomEndpoint": {
+      "defaultValue": "",
+      "type": "String",
+      "metadata": {
+        "description": "Enter the address of your private endpoint here (e.g. https://mycustomendpoint.cognitiveservices.azure.com/) if you are connecting with a private endpoint"
+      }
+    },
+    "ProfanityFilterMode": {
+      "defaultValue": "None",
+      "type": "String",
+      "allowedValues": [
+        "None",
+        "Removed",
+        "Tags",
+        "Masked"
+      ],
+      "metadata": {
+        "description": "The requested profanity filter mode."
+      }
+    },
+    "PunctuationMode": {
+      "defaultValue": "DictatedAndAutomatic",
+      "type": "String",
+      "allowedValues": [
+        "None",
+        "Dictated",
+        "Automatic",
+        "DictatedAndAutomatic"
+      ],
+      "metadata": {
+        "description": "The requested punctuation mode."
+      }
+    },
+    "AddDiarization": {
+      "defaultValue": false,
+      "type": "bool",
+      "metadata": {
+        "description": "A value indicating whether diarization (speaker separation) is requested."
+      }
+    },
+    "AddWordLevelTimestamps": {
+      "defaultValue": false,
+      "type": "bool",
+      "metadata": {
+        "description": "A value indicating whether word level timestamps are requested."
+      }
+    },
+    "TextAnalyticsKey": {
+      "defaultValue": "",
+      "type": "SecureString",
+      "metadata": {
+        "description": "The key for the Text Analytics subscription."
+      }
+    },
+    "TextAnalyticsEndpoint": {
+      "defaultValue": "",
+      "type": "String",
+      "metadata": {
+        "description": "The endpoint the Text Analytics subscription is associated with. If none is selected, no text analysis will be performed.."
```

Comment:
Can you update the description here?
It'd be great to give an example of the new format, like:

"The endpoint the Text Analytics subscription is associated with (format should be like https://{region}.api.cognitive.microsoft.com or https://{resourceName}.cognitiveservices.azure.com or similar). If empty, no text analysis will be performed."

Reply:
added Thanks

Reply:
added